src/components/store/OrderSummary/index.tsx

```diff
@@ -57,6 +49,12 @@ const OrderItemPreview = ({ item, orderStatus }: OrderItemPreviewProps) => {
           alt="Store item picture"
           fill
         />
+        {showFulfilled && item.fulfilled ? (
+          <div className={styles.badge}>{showNotFulfilled ? 'Picked up' : 'Already picked up'}</div>
```

Comment:
Thoughts on like a green for this instead of grey similar to how notAvailable is red?